layouts/fullpage.vue

```diff
@@ -3,6 +3,9 @@
     <v-main>
       <nuxt />
     </v-main>
+    <v-footer app color="rgba(0, 0, 0, 0)">
+      <locale-switcher :fab="false" />
```

Comment:
```suggestion
      <locale-switcher large />
```
First fab=false isn't needed for the prop to be false. Secondly, I'd enlarge the button a little bit